string                  label 						
geometry_msgs/Point     center
float32                 radius
float32                 radius_diff